app/views/docs/upgrade.phtml

```diff
@@ -68,8 +68,8 @@
 
 <p>Verify that the <b>STATUS</b> doesn't have any errors and all the <b>appwrite/appwrite</b> containers have version : <?php echo APP_VERSION_STABLE; ?></p>
 
-<h3><a href="/docs/upgrade#run-the-migration" id="migration">Running the Migration</a></h3>
-
+<h3><a href="/docs/upgrade#run-the-migration" id="migration">Running the Migration(Optional)</a></h3>
```

Comment:
Hey! Thanks for contributing to our docs :D 

We do already mention this is a notice on line 10: https://github.com/appwrite/docs/blob/d3c17289c36dbe4d701dadf7b7fbe0270d78db34/app/views/docs/upgrade.phtml#L10

What I suggest is the following:
- Removing the `(Optional)` from the heading and adding a notice 
- Remove the notice text from line 10.
- Add your note in a proper notice